Magento_Theme/styles/modules/_footer-menu.scss

```diff
@@ -0,0 +1,34 @@
+.footer-menu {
+    display: block;
+    width: 100%;
+    padding: 0;
+    margin: 15px 0;
+    list-style-type: none;
+    text-align: center;
+    @include mq($screen-m) {
+        display: flex;
+        flex-flow: row nowrap;
+        justify-content: center;
+        padding: 2px 0 2px 0;
+    }
+    .item {
+        display: block;
+        padding: 0;
+        margin: 0;
+        border-bottom: 1px solid $gray-lighter;
+        &:first-child {
+            border-top: 1px solid $gray-lighter;
+            @include mq($screen-m) {
+                border-top: 0;
+            }
+        }
+        @include mq($screen-m) {
+            border-bottom: 0;
+        }
+    }
+    a {
+        display: block;
```

Comment:
No this is not CMS block, it's default footer links and links here are generated by Magento\Framework\View\Element\Html\Link\Current which currently doesn't allow us to add class to link via XML